lib/rules/no-private-routing-service.js

```diff
@@ -93,6 +104,24 @@ module.exports = {
           context.report({ node, message: ROUTER_MICROLIB_ERROR_MESSAGE });
         }
       },
+
+      CallExpression(node) {
```

Comment:
Should we also check `OptionalCallExpression` for `owner?.lookup('router:main')`?

Reply:
I guess so! I updated to catch that. Of course, it would be pretty bad practice to use optional chaining since `lookup` should always exist on `owner`.

Reply:
This example would be the scenario where `owner` is `undefined`. For that, we'd need to handle `owner.lookup?.('router:main')`